episodes/05_Write_functions.md

```diff
@@ -118,6 +118,16 @@ Pkg.instantiate()
 
 ````
   Activating project at `~/projects/trebuchet`
+Precompiling project...
+[32m  ✓ [39m[90mOpenSSL_jll[39m
+[32m  ✓ [39m[90mOpenSSL[39m
+[32m  ✓ [39m[90mHTTP[39m
+[32m  ✓ [39m[90mWebSockets[39m
+[33m  ✓ [39m[90mWebIO[39m
+[33m  ✓ [39m[90mJSExpr[39m
+[33m  ✓ [39mTrebuchet
+  7 dependencies successfully precompiled in 416 seconds. 160 already precompiled.
+  [33m3[39m dependencies precompiled but different versions are currently loaded. Restart julia to access the new versions
```

Comment:
This broke the build of the webpage: https://github.com/carpentries-incubator/julia-novice/actions/runs/6642649083/job/18047884810.  Also, should the build be run also in PRs, to avoid having broken builds on `main`?

Reply:
would be nice, yeah, but we'd need to separate build from deploy for that, I guess